content/news/preview-20190606.md

```diff
@@ -0,0 +1,70 @@
+It has been quite a while since our last release, but not to worry, we are still going - mostly rewriting tons of code not visible to the outside eye. While not presenting a full playtest we want to take the time to introduce the community to some upcoming changes to look forward to.
+
+### Singleplayer savegames
+
+Probably the most wanted missing feature, the next version will include support for savegames in both skirmish and the campaigns. No more losing all progress to a stray dog eating your Tanya!
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-savegame-d2k.png" alt="The savegame load screen in D2k." width="600px">
+
+Longer missions can now be played in several takes instead of needing you to pull through in one go.
+</div>
+
+### Spectator UI and new Tiberian Dawn sidebar
+
+Our spectator UI has always been noted to be clunky and take too much space off the screen. With the next update we are happy to finally introduce a more slim and focused spectator UI.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-spectator-UI.png" alt="A new spectator UI." width="600px">
+
+The new transpart overlay is a huge quality of live upgrade for both casters and casual spectators.
+</div>
+
+Our second most visible update features yet another long standing UI rework (started in 2015!) that was just completed last month. The sidebar in Tiberian Dawn was completely overhauled and looks better than ever.
+
+### Polished building placement previews
+
+There is not much to say here, the images will tell the story. The tweaked placement preview makes it easier to see both the building being placed and any cells that are blocked by units or debris.
+
+On a side node for the modders reading this: With the coming update the core engine will also support placing building variants in your mod (if you like north and south facing refineries for example).
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-td-sidebar.gif" alt="The new Tiberian Dawn sidebar." width="600px">
+
+The new Tiberian Dawn UI includes a refreshed building placement overlay and brand new sidebars.
+</div>
+
+### Polished map previews
+
+The ingame UI was not the only thing getting an upgrade. We overhauled the map previews of the map chooser to look the same as ingame.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-map-previews.gif" alt="Overhauled map previews." width="600px">
+
+The new previews include the positions of forests and civilian structures.
```

Comment:
Maybe add `(before / after)` to indicate that it is animating?

Reply:
Done in the gif.